static/app/views/dashboards/widgetBuilder/buildSteps/sortByStep/sortBySelectors.tsx

```diff
@@ -131,7 +131,11 @@ export function SortBySelectors({
             label: sortDirections[value],
             value,
           }))}
-          value={values.sortDirection}
+          value={
+            REVERSE_ISSUE_FIELDS_SORT_LIST.includes(values.sortBy)
+              ? SortDirection.HIGH_TO_LOW
+              : values.sortDirection
+          }
```

Comment:
😭 haunting! We should really talk to the issues folks about supporting proper sorts...

That said, I think this logic should go into `useWidgetBuilderState`, under `case BuilderStateAction.SET_SORT:`. My sense is that this file is where all the fallback logic goes (e.g., if someone chooses the "Issues" dataset, it automatically chooses a Table visualization) so it'd make sure for it to drop down to the correct sort direction based on the field the user chose.

@nikkikapadia thoughts?

Plus, it's important that this _only_ happens for the Issues dataset, since users who are using Logs and Spans should still be able to set the sort direction for these fields 🙏🏻